app/pages/patient/patientinfo.js

```diff
@@ -289,7 +288,15 @@ var PatientInfo = React.createClass({
       return;
     }
 
-    return datetimeUtils.yearsOldText(birthday);
+    var yrsAgo = new Date().getFullYear()-new Date(birthday).getFullYear();
```

Comment:
Yeah, this isn't nuanced enough (nor is the similar code for diagnosis date).

Dates (not just years) need to be involved. If my birthday was March 9th instead of March 6th, this would still tell me today (March 8th) that I'm 31 years old, when I'd still only be 30...